src/CMakeLists.txt

```diff
@@ -628,3 +632,34 @@ if (NOT ANDROID)
     endif()
 endif()
 
+# ------------------------------------------------------------------------------
+# Protobuf library + proto generation
+# ------------------------------------------------------------------------------
+if (NOT ANDROID)
```

Comment:
Is it because for now you prefer to prototype without having to deal with it? Or is there an issue with protobuf<>android?

Reply:
Yes, there is an issue.
I use protoc to convert my proto to ***.pb.h and ***.pb.cc and it works fine when you build on the machine that is the target.
When cross-compiling for Android, protoc is built for android too... which is not really desirable.
I don't know how to fix that. :)

Reply:
the generated pb.h/cc files are not portable?

Reply:
it seems they are. I can rebuild in gradle with generated pb.h/cc. That's why I added them to the PR recently. Basically, it means someone touching the proto would need to build for Windows or Linux to generate the pb.h/cc, then switch to Android.

Reply:
Obsolete, as I moved to JSON.